docs/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### 5.2.0 Fixes
 
 - `[ListView]` - Added optional SohoListViewOptions arg to updated() function in listview.d.ts `PWP`  ([Pull Request 384](https://github.com/infor-design/enterprise-ng/pull/384))
+- `[Datagrid]` - Added hideable option to datagrid column typings.
```

Comment:
Add the pull request number, and your initials.

Reply:
OK, just updated.

Reply:
This should probably move up into the 5.3 fixes area